docs/development-guidelines/global-deployment.md

```diff
@@ -1,18 +1,32 @@
 # Global Deployment
 
-This document outlines a workaround for a known upstream bug handling global-standard type Azure OpenAI deployments (issue of resource deletion conflicts).
+This document outlines the migration guidance for transitioning to a global deployment SKU for GPT models in Azure OpenAI, as well as troubleshooting steps for handling resource deletion conflicts.
 
-## Issue
+## Migration Guidance
+
+To migrate to a global deployment SKU, follow these steps:
+
+1. **Add a Parameter for DeploymentSkuName**: Update your deployment configuration to include a parameter for `DeploymentSkuName`.
+
+2. **Default Sku to GlobalStandard**: Set the default value of the `DeploymentSkuName` parameter to `GlobalStandard` when possible. See the [global-standard-model-availability](https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models?tabs=python-secure#global-standard-model-availability) for more details.
+
+3. **Add Pre-Down Hooks**: Implement pre-down hooks to delete the model if needed. This ensures that the `azd down` process is not blocked and all resources are destroyed as expected. See the [troubleshooting section](#trouble-shooting) for more details.
```

Comment:
same, it will be better to be more clear on where to add the hooks, I think some owners may not very clear about azd hooks.

Reply:
I am concerned that we are giving instructions that are meant to help with a temporary workaround for the resource deletion conflict. I would recommend keeping the troubleshooting instructions clearly separated from the steps to migrate to the global deployment sku.

Reply:
I am torn. Is there an ETA for resolution of this issue? Since right now the intention is to migrate all templates that can be migrated, it is important the current workaround for the deletion problem is in place.

Reply:
We can achieve both the workaround and the clarity that it's a workaround. I'll propose a suggested edit for you to review.

Reply:
Thank you, I accepted it.